src/MyPinCreation/View/MyPinCreationInitiationController.cpp

```diff
@@ -54,7 +54,8 @@ namespace ExampleApp
 
             void MyPinCreationInitiationController::OnViewStateChangeScreenControl(ScreenControl::View::IScreenControlViewModel &viewModel, float &state)
             {
-                ScreenControl::View::Apply(m_viewModel, m_view);
+                //TODO: Uncomment to show My pins button
+                //ScreenControl::View::Apply(m_viewModel, m_view);
```

Comment:
Would it be easier just to not instantiate MyPinCreationInitiation/View module? Believe intent is that the modules should be functional but optionally instantiated/added to the view?